src/screens/ENSAssignRecordsSheet.js

```diff
@@ -194,7 +194,12 @@ export default function ENSAssignRecordsSheet() {
                 <Text align="center" color="accent" size="16px" weight="heavy">
                   {displayTitleLabel
                     ? lang.t(
-                        `profiles.${isEmptyProfile ? 'create' : 'edit'}.label`
+                        `profiles.${
+                          isEmptyProfile &&
+                          params.mode !== REGISTRATION_MODES.EDIT
```

Comment:
the reason why we only have `isEmptyProfile` here is because of this https://linear.app/rainbow/issue/RNBW-3413/from-the-intro-screen-when-you-select-one-of-your-existing-ens-names with this PR if i go to an empty ens it will say `Edit` even tho the records are empty, it should say `Create`

Reply:
Ah it wasn't causing a bug, but it was causing a flash of "Create your profile" upon editing a profile with records. Will see if I can eagerly cache it.

Reply:
ok makes sense!